src/python/pants/option/global_options.py

```diff
@@ -723,6 +723,13 @@ def register_bootstrap_options(cls, register):
             help="Directory to use for engine's local process execution sandboxing.",
             default=tempfile.gettempdir(),
         )
+        register(
```

Comment:
Maybe kill this until its used or use it now. That said, I'm proposing having a single setting for local execution cache storage (lmdb and named caches) elsewhere.

Reply:
It is now used: I think you might have observed a mix of two commits, because we do propagate this down by the end of the series (I think the github UI displays rebased commits in time order? not terribly helpful.)